### PR TITLE
Implement a popup dialog to upsell teams to new slack users

### PR DESCRIPTION
**Pull request by DerekBurch**

### CODE REVIEW

**Comment by andrewconner** [comment]
@cvializ

**Comment by DerekBurch** [comment]
hang on some search changes accidentally got added along with this...will remove them

File: kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js [hunk 1]
```diff
@@ -254,6 +255,12 @@ angular.module('kifi')
           extensionLiaison.triggerGuide();
           profileService.savePrefs({auto_show_guide: null});
           unregisterAutoShowGuide();
+        } else if (profileService.isFakeUser() && profileService.prefs.show_slack_create_team_popup) {
+          profileService.savePrefs({show_slack_create_team_popup: false});
+          modalService.open({
+            template: 'slack/newSlackUserTeamUpsellModal.tpl.html',
+            modalData: {}
+          });
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js:263]: Then, just to be a good citizen:
`unregisterAutoShowGuide();`

**Comment by cvializ** [comment]
lgtm!

File: kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js (hunk 1 above)
Comment (DerekBurch) [inline kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js:263]: Changed. Thanks

**Pull request merged**